### PR TITLE
Fix misspelled words

### PR DESCRIPTION
**Pull request by loverszhaokai**
Fix misspelled words found by [checkpatch.pl](https://github.com/loverszhaokai/JohnTheRipper/blob/fuzzing/run/checkpatch.pl)

### CODE REVIEW

File: src/gost.c [hunk 1]
```diff
@@ -24,7 +24,7 @@ extern unsigned rhash_gost_sbox_cryptpro[4][256];
  * Initialize algorithm context before calculaing hash
  * with test parameters set.
  *
- * @param ctx context to initalize
+ * @param ctx context to initailize
```
Comment (magnumripper) [inline src/gost.c]: Hey, these replace one typo with another!

Comment (loverszhaokai) [inline src/gost.c]: Sorry, my mistake.

**Comment by magnumripper** [comment]
This was an astonishing amount of typos, even in official AMD header stuff!

Fun fact of the day: You do get google hits for ADL_ERR_INVALID_DIPLAY_IDX but not for ADL_ERR_INVALID_DISPLAY_IDX! We should change it anyway of course. Just fix that "initailize" typo.

**Comment by loverszhaokai** [comment]
@magnumripper  Thanks for reminding. I should have **rechecked** again. :+1:

**Pull request merged**